app/cdap/components/ConnectionList/IconsStore/InfoGraph.tsx

```diff
@@ -0,0 +1,275 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+
+export const InfoGraph = () => {
+  return (
+    <svg
+      width="182"
+      height="146"
+      viewBox="0 0 182 146"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
```

Comment:
add a data-testid to this infography for testing

Reply:
Done